ReText/retext.qrc

```diff
@@ -0,0 +1,28 @@
+<RCC>
+    <qresource prefix="/icn_format">
+        <file>icons/format/light/blockquote.svg</file>
+        <file>icons/format/light/bold.svg</file>
```

Comment:
At least for bold, italics and underline (and probably also for images, bullets and numbering), it is possible to use theme icons:
- https://specifications.freedesktop.org/icon-naming-spec/latest/
- https://github.com/KDE/oxygen-icons/tree/master/22x22/actions

If the icons exist in the user theme, then it’s preferable to use those, otherwise we use the built-in ones.

But for the built-in ones, please use the existing mechanism (`getBundledIcon` function) instead of inventing a new one. This way you won't need the ugly `retext_rc.py` file.